src/annotator/guest.js

```diff
@@ -308,12 +310,9 @@ export default class Guest extends Delegator {
   destroy() {
     this._removeElementEvents();
     this.selections.unsubscribe();
-    this.adder.remove();
+    this.adderToolbar.remove();
 
-    this.element.find('.hypothesis-highlight').each((index, element) => {
-      $(element).contents().insertBefore(element);
-      $(element).remove();
-    });
+    removeAllHighlights(this.element[0]);
```

Comment:
One thing I've noticed is that this module has many references to `this.element[0]`; I wonder if it's worth assigning that reference to something a little more shorthand?

Reply:
My plan was to convert `this.element` to a regular DOM element soon. Then all uses of `this.element[0]` will become `this.element`.